eppo_metrics_sync/dbt_model_parser.py

```diff
@@ -0,0 +1,128 @@
+from collections import Counter
+from itertools import chain
+
+
+class DbtModelParser():
+
+    def __init__(self, model, dbt_model_prefix):
+        self.model = model
+        self.eppo_entities = []
+        self.eppo_timestamp = []
+        self.eppo_facts = []
+        self.eppo_properties = []
+        self.dbt_model_prefix = dbt_model_prefix
+    
+    def parse_columns(self):
```

Comment:
Two good visual rules of thumb for following SRP are 1. how big is the function? and 2. How indented is it?

By "indented" I mean that there are lines that start far to the right, which typically happens because of using nested loops or having a lot of conditional logic (if statements).

Highly indented code is also harder to read, so there's often an opportunity to split it up (better following SRP), which also makes the code more self-documenting. I'll leave some ideas below...